cpp/ps0001/main (3).cpp

```diff
@@ -1,27 +1,24 @@
-/******************************************************************************
-
-Welcome to GDB Online.
-GDB online is an online compiler and debugger tool for C, C++, Python, Java, PHP, Ruby, Perl,
-C#, OCaml, VB, Swift, Pascal, Fortran, Haskell, Objective-C, Assembly, HTML, CSS, JS, SQLite, Prolog.
-Code, Compile, Run and Debug online from anywhere in world.
-
-*******************************************************************************/
 #include <iostream>
 using namespace std;
 int main()
 {
-cout<<"please enter aNumber1?"<<endl;
-int MyNumber1;
-cin>>MyNumber;
-
-cout<<"please enter aNumber2?"<<endl;
-int MyNumber2;
-cin>>MyNumber2;
+  // prompting user for 3 numbers to add 1/2
+  cout<<"please enter aNumber1?"<<endl;
+  short MyNumber1;
+  cin>>MyNumber;
+  
+  cout<<"please enter aNumber2?"<<endl;
+  short MyNumber2;
+  cin>>MyNumber2;
+  
+  cout<<"please enter aNumber3?"<<endl;
+  short MyNumber3;
+  cin>>MyNumber3;
```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add